package.json

```diff
@@ -37,5 +37,9 @@
     "samples/*",
     "e2e-test-server",
     "scripts"
-  ]
+  ],
+  "overrides": {
+    "nx": "16.7.2",
+    "@nx/devkit": "16.7.2"
```

Comment:
These can be removed once node 14 is dropped. These packages broke support for node 14 in a minor version, this forces npm to install this version when generating a lockfile.

Reply:
Can be tracked in https://github.com/GoogleCloudPlatform/opentelemetry-operations-js/issues/630